source/userguide/active_learning.rst

```diff
@@ -0,0 +1,269 @@
+| tuning_level              | str        | Selecting classifiers to use for tuning, options: "domain" or "intent"           |
++---------------------------+------------+----------------------------------------------------------------------------------+
+| tuning_strategies         | List[str]  | List of strategies (heuristics) to use for tuning                                |
++---------------------------+------------+----------------------------------------------------------------------------------+
+
+4. **tuning_output**
+
++---------------------------+------------+----------------------------------------------------------------------------------+
+| Configuration Key         | Type       | Definition and Examples                                                          |
++===========================+============+==================================================================================+
+| save_sampled_queries      | bool       | Whether to save the queries sampled at each iteration                            |
++---------------------------+------------+----------------------------------------------------------------------------------+
+| aggregate_statistic       | str        | Metric to record aggregate classifier performance data in the output and plots,  |
+|                           |            | options: "accuracy", "f1_weighted", "f1_macro", "f1_micro"                       |
++---------------------------+------------+----------------------------------------------------------------------------------+
+| class_level_statistic     | str        | Metric to record class/tuning level performance data in the output and plots,    |
+|                           |            | options: "f_beta", "percision", "recall"                                         |
++---------------------------+------------+----------------------------------------------------------------------------------+
+
+5. **query_selection**
+
++---------------------------+------------+----------------------------------------------------------------------------------+
+| Configuration Key         | Type       | Definition and Examples                                                          |
++===========================+============+==================================================================================+
+| selection_strategy        | str        | Strategy (heuristic) to use for log selection                                    |
++---------------------------+------------+----------------------------------------------------------------------------------+
+| log_usage_pct             | float      | Percentage of the log data to use for selection                                  |
++---------------------------+------------+----------------------------------------------------------------------------------+
+| labeled_logs_pattern      | str        | Regex pattern to match log files if already labeled. For example, ".*log.*.txt"  |
```

Comment:
Should we mention here that annotated log files just mean that these files have been dumped appropriately in the hierarchy or is that understood?